### DISCUSSION TITLE
Quick tags pop up box

### DISCUSSION THREAD

**Discussion by Hananasaur** [AprilSylph/XKit-Rewritten#354] [Help Desk]
Hello! I'm a new user of Xkit-Rewritten, but a previous user of New XKit, and had enabled Quick Tags via the Google Chrome extension. I was wondering if it has a pop up box where I can quickly add tags to my post? 

New XKit used to have it near the bottom of the post controls where the like/reblog/edit buttons would be, or if I click on "Edit" a post, it would have been above the Post/Queue button. But now, I can't seem to find it with XKit Rewritten except when I'm on quick reblog and it shows my tag bundles.

It seems like Tumblr kind of changed their layout, too, so maybe that's why it's not showing anymore?

**Comment by AprilSylph** [comment]
If you've been opted into the new notes UI, it won't show up because the code looks for the edit button on posts to insert the Quick Tags button before. I don't have the new notes UI so I can't test any potential fix for this. My only recommendation at this stage is that you can try contacting Tumblr support to ask to be opted out of the test.

If you enable the beta post editor, you should still be able to use Quick Tags to use tag bundles when editing a post through Tumblr.

**Comment by Hananasaur** [comment]
Thanks for your quick response. It appears they've moved the "Edit" button on the top right menu now :\ 
![image](https://user-images.githubusercontent.com/95498431/145101645-0653f27b-b0e0-4287-8beb-a222130d07c2.png)

Anyway, I will see if I can get opted out of this test. Thank you again :)
